packages/sync/test/LocalDirectiveFilterLink.test.ts

```diff
@@ -26,11 +25,4 @@ describe("LocalDirectives", () => {
     expect(testLink.operations.length).equal(1);
     expect(!(hasDirectives(["onlineOnly"], testLink.operations[0].query)));
   });
-
-  it("ensures multiple directives do not exist after local link", () => {
```

Comment:
@wtrocki I left this one as it checks whether or not it works for mutliple directives - probably a good test anyway?